packages/react-popover/src/PopoverTrappingFocus.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Popover, PopoverTrigger, PopoverSurface } from './index';
+import { Button } from './utils.stories';
+import { ExampleContent } from './utils.stories';
```

Comment:
For CodeSandbox integration we should not have relative imports in stories. Do you plan to address this in this PR or separately?

Reply:
I threw up this PR before codesandbox story was finalized, will change it here